source/src/protocols/motifs/LigandDiscoverySearch.hh

```diff
@@ -182,6 +223,30 @@ private:
 	utility::vector1<core::conformation::ResidueOP> all_residues_;
 	// @brief residue index of protein pdb to attempt to place ligands off of
 	core::Size working_position_;
+	// @brief vector to hold list of all indices to investigate/use as anchor residues, used to set value of working_position_
+	utility::vector1<core::Size> working_positions_;
+
 	// @brief 3D matrix to represent voxelized copy of atoms in pose, used in clash check of placement for quick ruling out of bad placements
 	utility::vector1<utility::vector1<utility::vector1<bool>>> protein_representation_matrix_;
+
+	// @brief 3D matrix to represent voxelized copy of atoms in pose in a more geometrically-accurate means than protein_representation_matrix_, used in checking space fill analysis (of binding pocket)
+	utility::vector1<utility::vector1<utility::vector1<core::Size>>> protein_representation_matrix_space_fill_;
+
+	// @brief 1D vector to hold a list of residue indices to be considered in space fill function
+	utility::vector1<core::Size> target_residues_sf_;
+
+	// @brief 1D vector to hold a list of residue indices to be considered in residue-ligand contacts function
+	//utility::vector1<core::Size> target_residues_contact_;
+
+	// @brief a boolean value that controls the verbosity of text output while using the object
+	//default to be false, but can be made true by using the motifs::verbose flag; can also use set_verbose()
+	int verbose_;
+
+	// @brief a vector to act as a cipher for the space fill matrix with naming atoms with a unique name
+	//due to there being a limit in an atom name only having up to 4 characters and ther ebeign potential for a full representation matrix to have 1e5-1e6 atoms, using a base62 system for unique names
+	//base 62 allows over 14 million unique names
+	//represented as alphanumeric characters 0-9,a-z,A-Z
+	utility::vector1<std::string> base_62_cipher_{"0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n",
```

Comment:
It looks like that conversion could work in theory (and I like that it is in base 64 instead of 62). However, I do not like that "/" is in the character set, and fear it would not work for what I need. I am using this function for unique atom naming conventions that would be written into PDB files, and my understanding is that a slash could hurt parsing. If it was a more friendly character (- or _), that would be more welcome. Would a compromise be to make another function in  src/utility/Binary_Util.hh with a slightly modified character set that does not use the backslash? Perhaps even an option to use a custom 64 character set?

Reply:
Oh, if the provided character set doesn't work, then there isn't a need to combine the two.

You could potentially expand the current function to be flexible with respect to the character set, but that isn't needed if you don't think it's worth it.

I would add a comment as to why you're rolling your own, though.